client/lib/target_builder.dart

```diff
@@ -33,17 +36,33 @@ Target targetAll(GeneratedMessage instance) {
 }
 
 /// Creates a target which matches messages with the given IDs.
-Target targetByIds(GeneratedMessage instance, List<Any> ids) {
+Target targetByIds(GeneratedMessage instance, List<Object> ids) {
     var target = Target();
     target.type = _typeUrl(instance);
     var filters = TargetFilters();
     var idFilter = IdFilter();
-    idFilter.id.addAll(ids);
+    idFilter.id.addAll(ids.map(_idToAny));
     filters.idFilter = idFilter;
     target.filters = filters;
     return target;
 }
 
+Any _idToAny(Object rawId) {
```

Comment:
Won't we have something like `AnyPacker` and `Id`-packer?

It's just that a `target_builder` does not seem to be an appropriate place for such a common logic.